core/modules/wmgr/python/dbMgr.py

```diff
@@ -332,11 +343,11 @@ def createTable(dbName):
 
         # assume that we have correct SQL already
         try:
-            dbConn.useDb(dbName)
-            dbConn.execCommand0(schema)
-        except db.DbException as exc:
+            useDb(dbConn, dbName)
+            dbConn.execute(schema)
+        except sqlalchemyExc.OperationalError as exc:
             _log.error('Exception when creating table: %s', exc)
-            if exc.errCode() == db.DbException.TB_EXISTS:
+            if exc.args[0].startswith("(OperationalError) (1050,"):
```

Comment:
This looks very fragile, we should try to find a better way to identify this condition or at least hide it somewhere inside db package.